pyaedt/emit.py

```diff
@@ -9,289 +9,14 @@
 from pyaedt import generate_unique_project_name
 from pyaedt.application.AnalysisEmit import FieldAnalysisEmit
 from pyaedt.emit_core import EmitConstants
+from pyaedt.emit_core.results.results import Results
+from pyaedt.emit_core.results.revision import Revision
```

Comment:
Is the Revision class needed here, or only from within Results?

Reply:
As of now its needed by the main Emit class, but I haven't rearranged any of the class functions yet.